locale/en/about/working-groups.md

```diff
@@ -21,6 +21,7 @@ Core Working Groups are created by the
 * [Release](#release)
 * [Security](#security)
 * [Streams](#streams)
+* [Package Maintenance](#package-maintenance)
```

Comment:
The working groups are listed in alphabetical order so this should be moved up a few lines.

```suggestion
* [Package Maintenance](#package-maintenance)
* [Release](#release)
* [Security](#security)
* [Streams](#streams)
```